src/GitHub.App/ViewModels/Dialog/ForkRepositoryExecuteViewModel.cs

```diff
@@ -55,10 +56,14 @@ IRepositoryForkService repositoryForkService
 
         public IReactiveCommand<Repository> CreateFork { get; }
 
+        public IReactiveCommand<object> BackCommand { get; }
+
         public string Title => Resources.ForkRepositoryTitle;
 
         public IObservable<object> Done => CreateFork.Where(repository => repository != null);
 
+        public IObservable<object> Back => BackCommand.AsObservable();
```

Comment:
I don't think `Back` and `BackCommand` need to be separate here, you could just subscribe directly to `BackCommand` rather than exposing it in `Back`. The reason it's done like that with `Done` is that I didn't want to tie the `Done` signal to a `ReactiveCommand`: the dialog might be "done" when any number of things happen, some of which might not be easily represented as a command.